project.clj

```diff
@@ -1,4 +1,4 @@
-(def ps-version "5.2.0-SNAPSHOT")
+(def ps-version "5.2.0-master-SNAPSHOT")
```

Comment:
This pulls in the jar from a maven repo, right?

Reply:
a la https://artifactory.delivery.puppetlabs.net/artifactory/maven/puppetlabs/puppetserver/ or http://nexus.delivery.puppetlabs.net/content/repositories/releases/puppetlabs/puppetserver/

Reply:
No, this defines the version strings used when deploying to maven.

Reply:
oooh, my bad. My brain was in pe-puppet-server-extensions world.